includes/WikiPage.php

```diff
@@ -1967,7 +1967,7 @@ public function doUpdateRestrictions( array $limit, array $expiry, &$cascade, $r
 			);
 
 			wfRunHooks( 'NewRevisionFromEditComplete', array( $this, $nullRevision, $latest, $user ) );
-			wfRunHooks( 'ArticleProtectComplete', array( &$this, &$user, $limit, $reason ) );
+			wfRunHooks( 'ArticleProtectComplete', array( $this, &$user, $limit, $reason ) );
 		} else { # Protection of non-existing page (also known as "title protection")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)